leakpro/tests/input_handler/image_utils.py

```diff
@@ -0,0 +1,174 @@
+"""Utility functions for the image handler tests."""
+import os
+import pickle
+
+import numpy as np
+import torch.nn.functional as F  # noqa: N812
+from dotmap import DotMap
+from torch import Tensor, flatten, long, nn, save, stack, tensor
+from torch.nn import Module
+from torch.utils.data import TensorDataset
+from torchvision import transforms
+
+from leakpro.import_helper import Self
+
+STORAGE_PATH = "./leakpro/tests/input_handler"
+
+parameters = DotMap()
+parameters.epochs = 10
```

Comment:
You can consider setting these parameters from a config file. Then we can have different test_config files for different image datasets.